Makefile

```diff
@@ -0,0 +1,40 @@
+NAME ?= "KialiTestService"
+BINARY_NAME ?=kiali-test-service
+# Identifies the current build.
+# These will be embedded in the app and displayed when it starts.
+VERSION ?= 0.0.1.Final-SNAPSHOT
+COMMIT_HASH ?= $(shell git rev-parse HEAD)
+
+# Environment variables set when running the Go compiler.
+GO_BUILD_ENVVARS = \
+        GOOS=linux \
+        GOARCH=amd64 \
+        CGO_ENABLED=0 \
+
+DOCKER_NAME ?= kiali/kiali-test-service
+DOCKER_TAG ?= latest
+
+all: build
+
+build:
+	@echo Building...
+	${GO_BUILD_ENVVARS} go build \
+		-o ${GOPATH}/bin/${BINARY_NAME} -ldflags "-X main.name=${NAME} -X main.version=${VERSION} -X main.commitHash=${COMMIT_HASH}"
+
+docker-build:
```

Comment:
do we want to keep the make target names consistent with the other kiali make file? We use "docker" in swscore.

Reply:
I find 'docker-build' to be more clear what it is doing. Just 'docker' to me isn't clear enough.

Reply:
I suppose that just `docker` isn't clear enough because you are not thinking on the whole command which is `make docker` (making docker) :)